pyqtgraph/colormap.py

```diff
@@ -359,7 +359,13 @@ def getGradient(self, p1=None, p2=None):
         
         pos, color = self.getStops(mode=self.BYTE)
         color = [QtGui.QColor(*x) for x in color]
-        g.setStops(list(zip(pos, color)))
+        stops = zip(pos, color)
+        if hasattr(g, 'setStops'):
+            g.setStops(list(stops))
+        else:
+            # PySide6 has a missing setStops binding
```

Comment:
is this documented anywhere?  the method is referenced in the docs:

https://doc-snapshots.qt.io/qt6-dev/qgradient.html#setStops

Reply:
It's present in Qt6 but the binding is missing in PySide6. It could just be an unintended omission.
https://doc.qt.io/qtforpython/PySide6/QtGui/QGradient.html

In any case, hasattr(QtGui.QLinearGradient, 'setStops') is True for PySide2 but False for PySide6.